datasets/pcfg/data_generation.py

```diff
@@ -0,0 +1,124 @@
+""" Generating training and test files for iterative decoding on PCFG.
+"""
+
+import re
+import numpy as np
+
+# Specify local path to PCFG data in
+# https://github.com/i-machine-think/am-i-compositional/tree/master/data/pcfgset
+# Choose between iid (pcfgset), productivity and systematicity splits
+data_path = "/content/drive/MyDrive/preprocess_pcfg/pcfgset/data/"
+
+# Function that generates a list of intermediate tasks (including original source 
```

Comment:
Nit: docstrings for functions are usually inside of the function (see https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#s3.8.3-functions-and-methods )